src/sentry/static/sentry/app/views/eventsV2/miniGraph.tsx

```diff
@@ -103,4 +109,12 @@ class MiniGraph extends React.Component<Props> {
   }
 }
 
+const LoadingContainer = styled('div')`
```

Comment:
We have a similar component in `app/components/loading/loadingContainer` is there a way to adapt/use it here too?

Reply:
I opted to use `app/components/loading/loadingContainer`.